src/cli/commands/force/lightning/lwc/test/ui/mobile/run.ts

```diff
@@ -1,128 +1,93 @@
 import { Messages, SfError } from '@salesforce/core';
-import {
-    BaseCommand,
-    CommandLineUtils,
-    CommonUtils,
-    FlagsConfigType
-} from '@salesforce/lwc-dev-mobile-core';
-import * as childProcess from 'child_process';
-import util from 'util';
-import fs from 'fs';
-import path from 'path';
+import { BaseCommand, CommandLineUtils, CommonUtils, FlagsConfigType } from '@salesforce/lwc-dev-mobile-core';
 
-// Initialize Messages with the current plugin directory
-Messages.importMessagesDirectory(__dirname);
-
-// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
-// or any library that is using the messages framework can also be loaded this way.
-const messages = Messages.loadMessages(
-    '@salesforce/lwc-dev-mobile',
-    'test-ui-mobile-run'
-);
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile', 'test-ui-mobile-run');
 
 export class Run extends BaseCommand {
-    protected _commandName = 'force:lightning:lwc:test:ui:mobile:run';
-
-    public static readonly description =
-        messages.getMessage('commandDescription');
-
-    public static readonly examples = [
-        `sfdx force:lightning:lwc:test:ui:mobile:run --config '/path/to/wdio.conf.js'`,
-        `sfdx force:lightning:lwc:test:ui:mobile:run --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`,
-        `sfdx force:lightning:lwc:test:ui:mobile:run --config '/path/to/wdio.conf.js' --spec '/path/to/folderWithSpecs'`
-    ];
-
-    private static createError(stringId: string, ...param: any[]): SfError {
-        let msg = messages.getMessage(stringId);
-        if (param.length > 0) {
-            msg = util.format(msg, param);
-        }
-        return new SfError(msg, 'lwc-dev-mobile', Run.examples);
-    }
+    public static readonly summary = messages.getMessage('summary');
+    public static readonly examples = messages.getMessages('examples');
 
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     public static readonly flags = {
-        ...CommandLineUtils.createFlag(FlagsConfigType.Json, false),
-        ...CommandLineUtils.createFlag(FlagsConfigType.LogLevel, false),
+        ...CommandLineUtils.createFlag(FlagsConfigType.JsonFlag, false),
+        ...CommandLineUtils.createFlag(FlagsConfigType.LogLevelFlag, false),
         config: Flags.string({
-            description: messages.getMessage('configFlagDescription'),
             char: 'f',
-            required: true
+            description: messages.getMessage('flags.config.description'),
+            required: true,
+            validate: (config: string) => config && config.trim().length > 0
         }),
         spec: Flags.string({
-            description: messages.getMessage('specFlagDescription'),
-            char: 's',
-            required: false
+            description: messages.getMessage('flags.spec.description'),
```

Comment:
Do we need to specify `char: 's'` here?

Reply:
My PR is about migrating the codebase onto newer versions of Oclif and SFCore.... I don't want the PR to change the behavior of our CLI commands.

We used to have the `char: 's'` for this command which means that you could either pass in `--spec` flag or simply `-s`

I kept these as-is b/c like I said I don't want my PR to change command behavior.